src/lib/routing-tables.js

```diff
@@ -99,16 +115,54 @@ class RoutingTables {
   _removeRoute (ledgerB, ledgerC, connectorFromBToC) {
     this.eachSource((tableFromA, ledgerA) => {
       if (ledgerA !== ledgerB) return
-      tableFromA.removeRoute(ledgerC, connectorFromBToC)
+      return tableFromA.removeRoute(ledgerC, connectorFromBToC)
```

Comment:
good catch! that means expired route removal wasn't leading to the broadcast of broken links.  it's redundant as things stand (the connector being down is detected when we can't broadcast to them), but I'll make it work as intended.